src/gwkokab/models/utils/_lazyjointdistribution.py

```
@@ -0,0 +1,269 @@
+            self._support = _LazyConstraint(
+                *self.marginal_distributions,
+                dependencies=dependencies,
+                event_slices=self.shaped_values,
+            )
+        else:
+            self._support = support
+
+        self.partial_order = partial_order
+        self.dependencies = dependencies
+
+        super(LazyJointDistribution, self).__init__(
```

Ruff (0.14.2)

Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)